src/mon/PGMap.cc

```diff
@@ -3238,6 +3238,14 @@ void PGMap::get_health_checks(
 	summary += " reporting legacy (not per-pool) BlueStore omap usage stats";
       } else if (asum.first == "BLUESTORE_SPURIOUS_READ_ERRORS") {
         summary += " have spurious read errors";
+      } else if (asum.first == "BLUESTORE_SLOW_OP_ALERT") {
+        summary += " experiencing slow operations in BlueStore";
+      } else if (asum.first == "SLOW_DEVICE_STALLED_READ_ALERT") {
+        summary += " experiencing stalled read in slow device of BlueStore";
```

Comment:
same here - let's name it block device not slow device?

Reply:
Moreover - it looks like this summary addendum here brings no additional helpful information - we get pretty the same text from BlockDevice::collect_alert...